ui/src/dashboards/components/TemplateControlBar.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {SFC} from 'react'
+
+import classnames from 'classnames'
+import uuid from 'uuid'
+
+import Authorized, {EDITOR_ROLE} from 'src/auth/Authorized'
+import TemplateControlDropdown from 'src/dashboards/components/TemplateControlDropdown'
+
+interface TemplateValue {
+  value: string
+  selected?: boolean
+}
+
+interface Template {
+  id: string
+  tempVar: string
+  values: TemplateValue[]
+}
```

Comment:
`TemplateValue` and `Template` look like pretty standard types that could go in to a central types file